Sources/XCTest/Public/Asynchronous/XCTNSNotificationExpectation.swift

```diff
@@ -19,7 +19,7 @@ open class XCTNSNotificationExpectation: XCTestExpectation {
     /// - Returns: `true` if the expectation should be fulfilled, `false` if it should not.
     ///
     /// - SeeAlso: `XCTNSNotificationExpectation.handler`
-    public typealias Handler = (Notification) -> Bool
+    public typealias Handler = @Sendable (Notification) -> Bool
```

Comment:
Nope, not needed here. `@Sendable` here tells the compiler to inspect the body of the closure/function to make sure it's safe to send. `@preconcurrency` wouldn't help. (I'm not sure what effect it would have, but presumably it would suppress some real warnings, and we don't want that.)